### ISSUE TITLE
OpenMapTiles version

### ISSUE THREAD

**Issue by farfromrefug** [CartoDB/mobile-sdk#287]
I am not sure this is the right place but i don't know where else to put it.
I was wondering what version of OpenMapTiles you are using for carto vector tiles?
I would like to show park names but it does not seem to be there. OpenMapTiles added it in 3.9
https://github.com/openmaptiles/openmaptiles/pull/525

Thanks!

**Comment by mtehver** [comment]
@jaakla Perhaps Jaak can answer this?

**Comment by jaakla** [comment]
Frankly, I'm not sure how often the tile server data is updated, it may be older. The safest solution to have latest and greatest OMT tiles would be to purchase latest https://openmaptiles.com/downloads/tileset/osm/ package and self-host tiles.

In any case, do we have styling for Park names in style?

**Comment by farfromrefug** [comment]
@jaakla yep i know i can by packages but i am running an open source app right now. Don't really have the means to do so ...
Just think it would be nice to have the version you are using somewhere.

And no i dont think your current styles have park names.

**Comment by jaakla** [comment]
You can get maps directly from openmaptiles cloud server, and free plan is quite big already (100K maps per month). This has latest version and updated data. Their page https://cloud.maptiler.com/tiles/v3/ gives you unique URL for tiles , and they have even working CARTO mobile SDK code integration samples in same page!

**Comment by farfromrefug** [comment]
@jaakla thanks for letting me now. I tried and indeed i get parks names. Same style with Carto and i dont get them.
now the free plan is going too fast. After one day of dev i am already at 55% :s My point is it would be great for Carto to update to the latest version. Maybe on the next package update?
